app/views/renders/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="card-wrap">
+<img src=<%= user.photo %> alt="photo">
+<div class="card">
+<h2 class="card-title"><%= user.name%></h2> 
+<p class="card-para">number of posts: <%= user.posts_counter %> id: <%= user.id%> </p>
+</div>
+<li>
```

Comment:
- [ ] For us to visit the show user page, we need to link to it. Please create a link that leads to the show user page. I suggest you pick one of the element in this partial. For example, the image tag 👇🏼:

 ```ruby
 <%= link_to user do %>
   <%= image_tag user.photo, class: 'card-image', alt: 'photo' %>
 <% end %>
 ```